### PR TITLE
Hotel rework

### PR DESCRIPTION
**Pull request by Satile1**
# About The Pull Request

Adds a new reworked space hotel featuring more to do for the staff, a renovated bar and club and a new Manager role.

Contains materials for building Quantum Pads back and forth from the station, an engineering and service techfab and a smattering of resources. 

image

## Why It's Good For The Game

Current space hotel is a pretty sad and lonely ghostrole, and by and large a waste of space ruin budget when it spawns as it sees little traffic.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
A chain of space hotels has been bought out by Nanotrasen and will begin servicing nearby stations!

### CODE REVIEW

**Comment by Satile1** [comment]
Will need some config changes for this - put the original hotel on the spaceruins blacklist so they don't both spawn.  (and the non-splurt version of the syndicate listening station but that's a whole other thing)

**Comment by JerryWester** [comment]
> Engineering and service techfab

~~well, it's no omnilathe, but i certainly won't complain~~

**Comment by JerryWester** [comment]
So, I've noticed that the old space hotel still spawns in. Might wanna look into replacing it instead of adding a new one to fight over the space budget

**Comment by Satile1** [comment]
What I picked up from looking at similar projects was that a new map is better for modularity. Keeping all new additions separate from the citadel code..

The previous hotel can certainly be stopped from spawning - but it requires a config change by the host.

**Comment by BongaTheProto** [comment]
> What I picked up from looking at similar projects was that a new map is better for modularity. Keeping all new additions separate from the citadel code..
> 
> The previous hotel can certainly be stopped from spawning - but it requires a config change by the host. 

I'm pretty sure it needs to be deactivated through code

**Comment by JerryWester** [comment]
AFAIK, it would pretty much be as simple as:
```
/datum/map_template/ruin/space/spacehotel
    unpickable = TRUE
```
In order to disable

File: code/datums/ruins/space.dm [hunk 1]
```diff
@@ -148,6 +148,7 @@
 	suffix = "spacehotel.dmm"
 	name = "The Twin-Nexus Hotel"
 	description = "An interstellar hotel, where the weary spaceman can rest their head and relax, assured that the residental staff will not murder them in their sleep. Probably."
+	unpickable = TRUE	//Splurt Change
```
Comment (BongaTheProto) [inline code/datums/ruins/space.dm:151]: could just add this modularly tbh

**Review by BongaTheProto — CHANGES_REQUESTED** [review]

File: code/datums/ruins/space.dm (hunk 1 above)
Comment (Satile1) [reply code/datums/ruins/space.dm:151]: Not sure how to go about that in this case - could you suggest a method?

Comment (BongaTheProto) [reply code/datums/ruins/space.dm:151]: > Not sure how to go about that in this case - could you suggest a method?

create a modular_splurt/code/datums/ruins/space.dm (if there isn't one already) and add something like

```
/datum/map_template/ruin/space/spacehotel
	unpickable = TRUE
```

**Pull request merged**